tasmota/tasmota_template_legacy.h

```diff
@@ -87,6 +87,8 @@ enum LegacyUserSelectablePins {
   GPI8_LED4_INV,
   GPI8_MHZ_TXD,        // MH-Z19 Serial interface
   GPI8_MHZ_RXD,        // MH-Z19 Serial interface
+  GPI8_CM11_TXD,       // CM110x Serial interface
+  GPI8_CM11_RXD,       // CM110x Serial interface
```

Comment:
Changes in this file are not supported. Pls remove any changes. See top of this file.

Reply:
Done.